doc/sphinx/source/quickstart/installation.rst

```diff
@@ -59,20 +59,20 @@ by running (Linux example):
     bash Miniconda3-latest-Linux-x86_64.sh
 
 and follow the instructions on your screen.
-Immediately update conda after installing it:
+Immediately update mamba after installing it:
 
 .. code-block:: bash
 
-    conda update --name base conda
+    mamba update --name base mamba
```

Comment:
no, these should not change, conda needs to be updated, then installing mamba from conda-forge insures that mamba is at the latest version; if they need to update conda then they can do `conda update -c conda-forge mamba`